widgetssdk/src/main/java/com/glia/widgets/config/components/Layer.java

```diff
@@ -0,0 +1,43 @@
+package com.glia.widgets.config.components;
+
+import androidx.annotation.Nullable;
+
+import com.glia.widgets.config.components.base.DpSize;
+import com.google.gson.annotations.SerializedName;
+
+public class Layer {
```

Comment:
Will probably need to remove the existing LayerConfiguration because it has the same purpose.

Reply:
I'm not sure. `LayerConfiguration` is a `public` class, and may be used by integrators.